### ISSUE TITLE
Cannot run app from root folder of repo

### ISSUE THREAD

**Issue by willu47** [KTH-dESA/unescap#4]
When running `python dashboard/app.py` I get the following error:

```python
Traceback (most recent call last):
  File "dashboard/app.py", line 14, in <module>
    input_tfec = pd.read_excel(os.path.join(folder, 'TFEC.xlsx'))
  File "/miniconda3/envs/unescap/lib/python3.6/site-packages/pandas/util/_decorators.py", line 208, in wrapper
    return func(*args, **kwargs)
  File "/miniconda3/envs/unescap/lib/python3.6/site-packages/pandas/io/excel/_base.py", line 310, in read_excel
    io = ExcelFile(io, engine=engine)
  File "/miniconda3/envs/unescap/lib/python3.6/site-packages/pandas/io/excel/_base.py", line 819, in __init__
    self._reader = self._engines[engine](self._io)
  File "/miniconda3/envs/unescap/lib/python3.6/site-packages/pandas/io/excel/_xlrd.py", line 21, in __init__
    super().__init__(filepath_or_buffer)
  File "/miniconda3/envs/unescap/lib/python3.6/site-packages/pandas/io/excel/_base.py", line 359, in __init__
    self.book = self.load_workbook(filepath_or_buffer)
  File "/miniconda3/envs/unescap/lib/python3.6/site-packages/pandas/io/excel/_xlrd.py", line 36, in load_workbook
    return open_workbook(filepath_or_buffer)
  File "/miniconda3/envs/unescap/lib/python3.6/site-packages/xlrd/__init__.py", line 111, in open_workbook
    with open(filename, "rb") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'Data/TFEC.xlsx'
```

**Comment by camiloramirezgo** [comment]
So if I understand well, the bug appears when running from the root folder, that is from the unescap folder, right? then should I change the root folder in the app to that one or have it look in both folders?
The thing is that when we push to Heroku, I guess we will send only the dashboard folder, thus it makes sense to me to keep the root for the visualisation app as the dashboard folder.

**Comment by willu47** [comment]
I've made a fix in a7587e3 which builds an absolute path to the `Data` folder from the location of the `app.py` file itself.

**Comment by camiloramirezgo** [comment]
Excellent looks good, I see you also added the Procfile for Heroku, thanks! Should we merge your branch already?
